packages/containers-shared/src/knobs.ts

```diff
@@ -2,3 +2,13 @@
 export const getCloudflareContainerRegistry = () => {
 	return process.env.CLOUDFLARE_CONTAINER_REGISTRY ?? "registry.cloudflare.com";
 };
+
+/**
+ * Given a container image that is a registry link, this function
+ * returns true if the link points the Cloudflare container registry
+ * (defined as per `getCloudflareContainerRegistry` above)
+ */
+export function isCloudflareRegistryLink(image: string) {
```

Comment:
is this function actually doing what we want? `process.env.CLOUDFLARE_CONTAINER_REGISTRY` could technically be set to anything, so I'm wondering if the check here shouldn't be `image.includes("registry.cloudflare.com")` instead 🤔